### DISCUSSION TITLE
Icon for SLA state in Flex queue activity details

### DISCUSSION THREAD

**Discussion by Fazle-Jahangir** [twilio-labs/paste#3175] [General] [answered]
[Redacted]

Non-public information

**Comment by TheSisb** [comment, marked as answer]
Hi @Fazle-Jahangir,

We have the [Process Status pattern](https://paste.twilio.design/patterns/status#process-status) that we recommend for situations like these. Would that be a viable path forward for you?

**Reply by Fazle-Jahangir** [reply]
It could work yes. I'll do some exploration with it and get feedback from the team. Thanks
